src/components/Auth/Forms/login.tsx

```diff
@@ -58,12 +64,10 @@ const LoginForm = () => {
 
   return (
     <div>
-      <div className="w-full">
-        <SignInOAuthButton
-          strategy="oauth_google"
-          label="Continue with Google"
-          icon={googleIcon}
-        />
+      <div className="relative flex justify-center align-center pt-5 pb-4 gap-x-0.5">
```

Comment:
Couple things
- `align-center` is not a standard Tailwind CSS class. 
- The class `gap-x-0.5` provides very minimal horizontal space between buttons, doesn't really seem to add much.